web/src/i18n/locales/ru.json

```
@@ -3166,6 +3166,135 @@
     "默认折叠侧边栏": "Сворачивать боковую панель по умолчанию",
     "默认测试模型": "Модель для тестирования по умолчанию",
     "默认用户消息": "Здравствуйте",
-    "默认补全倍率": "Default completion ratio"
+"默认补全倍率": "Default completion ratio",
```

Biome (2.4.4)

[error] The key 默认补全倍率 was already declared.

(lint/suspicious/noDuplicateObjectKeys)